### DISCUSSION TITLE
Ploting the Graph representation

### DISCUSSION THREAD

**Discussion by jesusfbes** [microsoft/graphrag#427] [General]
Is there a prefered way to represent/plotting the resulting graph ?

**Comment by stevetru1** [comment]
The easiest way to do that is to use graphml, and to create that you can edit your settings.yaml file like so (down at the bottom):
```
snapshots:
  graphml: true
```

As long as you still have your caches from the prior runs you should be able to just re-run the index and it will run through each step creating those graphml files for you.

As for visualizations, our team tends to use gephi but that's more of a preference question I think.

**Reply by kouskouss** [reply]
If I re-run the indexing process with the updated setting to create graphml files, will it re-index the files from the beginning in the same folder and renew the Parquet files as well?

**Reply by nikhilmaddirala** [reply]
> If I re-run the indexing process with the updated setting to create graphml files, will it re-index the files from the beginning in the same folder and renew the Parquet files as well?

@stevetru1 I have the same question!

**Reply by realguoqiang** [reply]
@nikhilmaddirala @kouskouss In my test, there is no re-index.

**Comment by jesusfbes** [comment]
I works perfectly. Thank you very much.
